### PR TITLE
Mintable ERC20 Example Tunnel Contracts Fixes

### PR DESCRIPTION
**Pull request by DhairyaSethi**
Reference: #41

### CODE REVIEW

File: contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol [hunk 1]
```diff
@@ -4,17 +4,29 @@ pragma solidity ^0.8.0;
 import {FxBaseChildTunnel} from "../../tunnel/FxBaseChildTunnel.sol";
 import {Create2} from "../../lib/Create2.sol";
 import {Ownable} from "../../lib/Ownable.sol";
```
Comment (dev1644) [inline contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol]: Not need to import it.

File: contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol [hunk 2]
```diff
@@ -4,17 +4,29 @@ pragma solidity ^0.8.0;
...
-import {FxERC20} from "../../tokens/FxERC20.sol";
+import {FxMintableERC20} from "../../tokens/FxMintableERC20.sol";
```
Comment (dev1644) [inline contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol]: Minor optimisation: Interface can be used instead of the Contract

**Review by dev1644 — CHANGES_REQUESTED** [review]

File: contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol (hunk 1 above)
Comment (DhairyaSethi) [reply contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol]: bf4f074

File: contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol (hunk 2 above)
Comment (DhairyaSethi) [reply contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol]: [a2906a0](https://github.com/fx-portal/contracts/pull/43/commits/a2906a0abb03b8fabd9a41d530ee1a2d914187f1)

**Review by QEDK — APPROVED** [review]
LGTM.

**Review by dev1644 — APPROVED** [review]

**Pull request merged**